airbyte-scheduler/src/test/java/io/airbyte/scheduler/ScheduleJobPredicateTest.java

```diff
@@ -82,24 +83,28 @@ public void testScheduleNotReady() {
     assertFalse(scheduleJobPredicate.test(Optional.of(job), SCHEDULE));
   }
 
+  // use Mode.EXCLUDE so that when new values are added to the enum, these tests will fail if that
```

Comment:
ideally when i changed the enum, these tests would have failed alerting me that a change was needed. now using this exclude trick i think they should do that.